management/status_checks.py

```diff
@@ -646,7 +646,7 @@ def preferred_ds_order_func(ds_suggestion):
 		output.print_line("Option " + str(i+1) + ":")
 		output.print_line("----------")
 		output.print_line("Key Tag: " + ds_suggestion['keytag'])
-		output.print_line("Key Flags: KSK")
+		output.print_line("Key Flags: KSK (256)")
```

Comment:
I apologize for the noise. I should have made this

```
Key Flags: 256 / KSK
```

The match the style of the following lines.

Reply:
Eh it's ok, we can leave it.